src/app/create-resource/create-resource.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="new-resource-form">
+  <h2> Create a new resource</h2>
+  <form>
+    <div class="form-group">
+      <label for="projectName" class="form-label"> Project Name </label>
+      <input type="text" disabled id="projectName" class="form-control" placeholder="{{projectName}}">
+    </div>
+    <div class="form-group">
+      <label for="name" class="form-label">Resource Name</label>
+      <input #resourceName placeholder="My New Resource" required minlength="3" type="text" class="form-control" ngModel name="name" #name="ngModel" id="name">
+    </div>
+    <div class="form-group">
+      <label for="rest" class="form-label"> Action </label>
+      <input #rest type="url" class="form-control" id="rest" placeholder="DELETE">
```

Comment:
The user shouldn't be able to insert here whatever he\she likes, there is a closed list of options that could be set as value here. Which type of input would be more appropriate?